### PR TITLE
vo_gpu: libplacebo: require v2.72.0

### PR DESCRIPTION
**Pull request by haasn**
It's about a year old, and packaged pretty much everywhere that bothers
to package libplacebo at all. Older versions are unlikely to be
prevalent and/or useful.

Starting with v2.72.0, libplacebo validates all of its parameters
internally and turns them into function failures. Doing it twice is
awfully redundant, so we can drop the parameter validation.

Also allows us to drop some preprocessor macros.

Read this before you submit this pull request:
https://github.com/mpv-player/mpv/blob/master/DOCS/contribute.md

Reading this link and following the rules will get your pull request reviewed
and merged faster. Nobody wants lazy pull requests.

### CODE REVIEW

**Review by philipl — APPROVED** [review]

**Comment by jeeb** [comment]
So did a quick check:
1. Ubuntu 20.04 LTS has 1.7.0 https://packages.ubuntu.com/focal/libplacebo-dev
2. Ubuntu 20.10 (shorter term non-LTS) has 2.72.0 https://packages.ubuntu.com/groovy/libplacebo-dev
3. Fedora 33 (from October) has 2.72.2 https://bodhi.fedoraproject.org/updates/?packages=libplacebo
4. Debian 10 matches Ubuntu 20.04
5. Debian 11 matches Ubuntu 20.10
6. Msys2 (rolling) has 3.104.0 (looking at a recent Appveyor log @ https://ci.appveyor.com/project/mpvplayer/mpv/builds/37898498/job/85a6qgbj23cpswj3?fullLog=true )

So basically, I think I am all for requiring newer versions of liblacebo, but just noting that _"Older versions are unlikely to be prevalent and/or useful"_ is maybe a bit underestimated due to the current Ubuntu LTS (from **April 2020**) not having a version starting with 2.x :) .

Just means that any CIs which do not have a new enough libplacebo would need to be bumped after this.

**Review by jeeb — APPROVED** [review]
Code-wise I have no problem with this, but I think the commit message could be reworded a bit, since clearly various places that are:

1. packaging libplacebo
2. have a large user base

Still have 1.7.0 series packaged, as much as you might not like that. Otherwise, thanks for putting this up to review, my only qualms were regarding that one bit in the commit message.

**Comment by haasn** [comment]
Thanks for looking into this. Updated the wording and rebased.

**Review by jeeb — APPROVED** [review]
LGTM.
